openaddr/cache.py

```diff
@@ -352,6 +359,7 @@ def field_names_to_request(cls, source_config):
                     fields.add(v)
 
         if fields:
+            # Remove any blank or None values
             return list(filter(None, sorted(fields)))
```

Comment:
Hi @iandees, on this line I think `sorted()` will still error if `fields` has a `None` in it (that's the original bug I hit yesterday, though as you discovered it was masking an issue with the `constant` function).

Let me know if I'm misunderstanding but I think this is the call order that will guarantee `sorted()` is never passed a `None`:

```py
return list(sorted(filter(None, fields)))
```

Reply:
Yep you're right. I think I wasn't running into this because I changed the code that generates the `fields` variable to not include any `None`s, but it's still worth fixing. I'll put together another PR.